src/System.ComponentModel.Composition/ref/System.ComponentModel.Composition.csproj

```diff
@@ -11,4 +11,13 @@
     <Compile Include="System.ComponentModel.Composition.cs" />
     <Compile Include="System.ComponentModel.Composition.Forwards.cs" />
   </ItemGroup>
+
+  <!-- In net461 we want to use the assembly that is part of the targetting pack, but BinPlace will replace it with
+  the live-built netstandard configuration, so we need to not binplace in those TFMs -->
+  <Target Name="RemoveNetFxBinPlacing" BeforeTargets="GetBinPlaceConfiguration">
```

Comment:
@safern @ericstj This is one of the cases where BinPlace was using the netstandard configuration and overriding the one comming from the desktop targeting pack.

Reply:
Instead of this, try making https://github.com/joperezr/corefx/blob/10df782d12ec1c1123f2233f81206899c6ef07f4/src/System.ComponentModel.Composition/ref/Configurations.props#L5 be `_net461`

Reply:
Ok I'll try that.